drivers/volume/csi/csi.go

```diff
@@ -552,8 +552,8 @@ func (c *csi) uploadObject(
 		return err
 	}
 
-	if backupLocation.Location.EncryptionKey != "" {
```

Comment:
Lets fail the uploadObject if `backupLocation.Location.EncryptionKey` is set. Return an error message
```
EncryptionKey is deprecated, use EncryptionKeyV2 instead.
```

Reply:
Fixed